metadata.rb

```diff
@@ -4,7 +4,7 @@
 license          'Apache-2.0'
 description      'Installs and configures Kerberos V authentication'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '2.3.0'
+version          '2.3.1'
```

Comment:
Don't bump the version, please. I'll do that when I release a new version to Supermarket.

Reply:
apologies, wasn't sure if I needed to add that or not.